### PR TITLE
Reorganized directory structure.

### PR DESCRIPTION
**Pull request by td-extreme**
Removed no longer used interfaces.

Moved interfaces to the folder of the class that owns / uses that
interface.

Moved exceptions into the folder of class the throws that exception.

Changed Ihanler to IHandler to conform to the naming style of the rest of my interfaces.

### CODE REVIEW

File: src/main/java/httpConnectionProcessing/IHandlerRouter.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.td.HttpServer;
+
+public interface IHandlerRouter {
```
Comment (td-extreme) [inline src/main/java/httpConnectionProcessing/IHandlerRouter.java:3]: This file isn't actually new. For some reason it didn't recognize this as a rename.

File: src/main/java/interfaces/IHandlerRouter.java [hunk 2]
```diff
@@ -1,6 +0,0 @@
-package com.td.HttpServer;
-
-public interface IHandlerRouter {
-  public Ihandler selectHandlerBadRequest();
```
Comment (td-extreme) [inline src/main/java/interfaces/IHandlerRouter.java:4]: This file wasn't deleted. See comment above about it not being recognized by git as a rename.

**Comment by dadarek** [comment]
Looks good. Ship it mister.

**Comment by td-extreme** [comment]
Wilco

**Pull request merged**